main.go

```diff
@@ -39,7 +33,10 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
-	//+kubebuilder:scaffold:imports
```

Comment:
Could you please keep this comment? Just move it to just before `"github.com/open-cluster-management/governance-policy-spec-sync/controllers/sync"`.